README.md

```diff
@@ -96,6 +99,8 @@ Then run one of the language-based sample apps below:
 * DotNET (coming soon)
 * Go (coming soon)
 
+Each of these sample apps works well with the [recipes](recipes) listed below.
```

Comment:
Are you trying to link to the recipes folder, or the header below?

Reply:
in this I was going right to the recipes folder, but I can make it go to the header

Reply:
Just making sure that was the intention